src/daemon/daemon.cpp

```diff
@@ -135,8 +137,37 @@ auto make_cloud_init_meta_config(const std::string& name)
     return meta_data;
 }
 
+auto make_cloud_init_network_config(const mp::NetworkInterface& default_interface,
+                                    const std::vector<mp::NetworkInterface>& extra_interfaces)
+{
+    YAML::Node network_data, interfaces_data;
+
+    network_data["version"] = "2";
+
+    std::string name = "default";
+    network_data["ethernets"][name]["match"]["macaddress"] = default_interface.mac_address;
+    network_data["ethernets"][name]["dhcp4"] = true;
+    network_data["ethernets"][name]["wakeonlan"] = "true";
```

Comment:
Oh, right, Qt uses `QString`'s. Corrected.
I don't understand however what do you mean with "against the default".

Reply:
OK, let me try to clarify:

- why is "wakeonlan" set to a string ("true") while "dhcp4" set a boolean? Couldn't it be just `true` as well?
- why did you choose to enable wakeonlan? (the default is for it to be disabled)

Reply:
Ok, thanks for the clarification, now I understand. `"true"` was changed to `true`.
About `wakeonlan`, I enabled it to allow the instance to be resumed upon connection. But well, this is easily set up once booted, so I'll be ok to remove it.

Reply:
I don't know, I am not sure what actual use case this has in mind, and it could cause surprises (VMs seemingly starting spontaneously). Perhaps I could be convinced of the advantage, but by default, it feels safer to align with the netplan's default, wouldn't you say?

Reply:
Yep, better stick to the default.

Reply:
Yep, better stick to the default. Removed `wakeonlan`.